src/main/java/com/maybe/maybe/controller/InvoiceController.java

```diff
@@ -0,0 +1,71 @@
+package com.maybe.maybe.controller;
+
+import com.maybe.maybe.dto.InvoiceDTO;
+import com.maybe.maybe.entity.Invoice;
+import com.maybe.maybe.service.InvoiceService;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import java.time.LocalDateTime;
+
+@RestController
+@RequestMapping("/api")
+public class InvoiceController {
+
+    private InvoiceService invoiceService;
+
+    public InvoiceController(InvoiceService invoiceService) {
+        this.invoiceService = invoiceService;
+    }
+
+    @GetMapping("/invoices/")
```

Comment:
i think the last slash should be absent 
https://restfulapi.net/resource-naming/